README.md

```diff
@@ -22,90 +22,57 @@
 
-The Payjoin Dev Kit `payjoin` library implements both [BIP 78 Payjoin V1](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) and [BIP 77 Payjoin V2](https://github.com/bitcoin/bips/blob/master/bip-0077.md).
+Find the description of each crate below.
 
-### `payjoin-cli`
+### [`payjoin`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin)
 
-The [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli) crate performs no-frills Payjoin as a reference implementation using Bitcoin Core wallet.
+The main Payjoin Dev Kit library which provides tools for implementing both Payjoin V2 (Asynchronous) and V1 (Synchronous). For Payjoin V2, the library also contains Payjoin session persistence support and IO utilities for interacting with OHTTP relays.
 
-### `payjoin-directory`
+**Disclaimer: This crate has not been reviewed by independent Rust and Bitcoin security professionals (yet). Use at your own risk.**
 
-The [`payjoin-directory`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-directory) crate implements the Payjoin Directory store-and-forward server required for Payjoin V2's asynchronous operation.
+### [`payjoin-cli`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-cli)
 
-### `payjoin-test-utils`
+A CLI tool which performs no-frills Payjoin. It is a reference implementation of the Payjoin Dev Kit which uses a Bitcoin Core wallet.
 
-The [`payjoin-test-utils`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-test-utils) crate provides commonly used testing fixtures such as a local OHTTP relay and payjoin directory, bitcoind node and wallets, and official test vectors.
+### [`ohttp-relay`](https://github.com/payjoin/rust-payjoin/tree/master/ohttp-relay)
 
-### `payjoin-ffi`
+A Rust implementation of an Oblivious HTTP (OHTTP) relay resource.
 
-The [`payjoin-ffi`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-ffi) crate provides language bindings that expose the Rust-based Payjoin implementation to various programming languages.
+**Disclaimer: Both this crate and the [IETF paper](https://ietf-wg-ohai.github.io/oblivious-http/draft-ietf-ohai-ohttp.html) are undergoing active revision. Use at your own risk.**
 
-### Disclaimer ⚠️ WIP
+### [`payjoin-directory`](https://github.com/payjoin/rust-payjoin/tree/master/payjoin-directory)
 
-**Use at your own risk. This crate has not yet been reviewed by independent Rust and Bitcoin security professionals.**
+A reference implementation for a Payjoin directory which stores and forwards HTTP client messages between the sender and the receiver to allow for asynchronous Payjoin transactions. Payjoin V2 clients encapsulate requests using [Oblivious HTTP (OHTTP)](https://www.ietf.org/rfc/rfc9458.html) which allows them to make payjoins without the directory being able to link payjoins to specific client IPs.
```

Comment:
```suggestion
A reference implementation for a Payjoin Directory which stores and forwards messages between the sender and the receiver to allow for Async Payjoin transactions. Async Payjoin clients make requests to the directory using [Oblivious HTTP (OHTTP)](https://www.ietf.org/rfc/rfc9458.html) which prevents the directory from being able to link payjoins to specific client IP addresses.
```

- Payjoin Directory is capitalized as a proper noun everywhere the words are used together in the BIP. I suggest keeping consistent.
- HTTP client messages aren't formatted, special payloads are. "messages" is sufficient imo
- Async Payjoin is the name of BIP 77.
- The protocol making use of both encapsulation and the relay prevents the linking, not just encapsulation